app/components/agenda/printable-agenda/list-section/item-group/item/document-list.hbs

```diff
@@ -3,9 +3,6 @@
     <li>
       <span class="auk-o-flex--inline auk-o-flex--vertical-center">
         <span>({{piece.name}})</span>
-        {{#if piece.confidential}}
-          <Auk::Icon @name="lock-closed" @skin="danger" class="auk-u-ml" />
-        {{/if}}
```

Comment:
It's ok to remove the confidential indicator, but maybe they want to see an indication for documents with access-level 'Intern secretarie' or 'Ministerraad' instead?

Reply:
Indicator → access level pill. Is that okay? Or do we want to keep it icon-only, like the confidentiality lock? Though I think it might get confusing if we mix and match visual indicators for the same concept.

Reply:
Pill is what I meant :+1:  Reusing lock icon will be confusing indeed.